include/bout/petsclib.hxx

```diff
@@ -47,6 +47,8 @@ class PetscLib;
 
 #ifdef BOUT_HAS_PETSC
 
+#include "options.hxx"
```

Comment:
Could this just be a forward declaration? 

I'm not sure on our current policy regarding favouring forward declarations or not.

Reply:
Forward declaration should be ok here